style.css

```diff
@@ -525,3 +525,562 @@ img:hover {
   overflow-y: scroll;
   -webkit-overflow-scrolling: touch;
 }
+
+@media screen and (min-width: 786px) {
+  .wrapper {
+    width: 100%;
+    background-color: #fff;
+  }
+
+  body {
+    width: 100%;
+  }
+
+  .nav-links {
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .hambuger {
+    display: none;
+  }
+
+  .nav-links span a {
+    text-decoration: none;
+    padding: 10px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .header-wrapper {
+    background-image: url(./Images/Illustration-header.png);
+    background-position: right top;
+    color: white;
+    background-position-y: 3.5rem;
+    background-size: 35%;
+    height: 27.9%;
+    margin-bottom: 5%;
+  }
+
+  .header-section {
+    align-items: center;
+  }
+
+  .first-section {
+    display: flex;
+    flex-direction: row-reverse;
+    width: 39.5%;
+    background-color: #1c1a19;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .header-intro {
+    margin-left: 10%;
+  }
+
+  .header-text {
+    font-size: 72px;
+    line-height: 113%;
+    color: #ff6b00;
+    margin-block-start: 0;
+    margin-block-end: 0;
+    margin-left: 14px;
+    margin-top: 16rem;
+  }
+
+  .header-smaller-text {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 35px;
+    margin-block-start: 0;
+    margin-block-end: 0;
+    line-height: 44px;
+    padding-top: 2px;
+    margin-top: 0;
+  }
+
+  .header-paragraph {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    line-height: 32px;
+    margin-bottom: 15.6rem;
+  }
+
+  .header-icons {
+    display: flex;
+    padding-left: 10px;
+  }
+
+  .header-icons ul {
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 0;
+  }
+
+  .header-icons ul li {
+    padding-left: 20px;
+    padding-bottom: 10px;
+  }
+
+  .work-section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .workSection1 {
```

Comment:
- [ ] Please center this section because it appears more on the left side of the page. Try as much as possible to stick on [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)